src/main/java/core/basesyntax/DifferentColor.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public enum DifferentColor {
```

Comment:
Consider renaming the enum to 'Color' instead of 'DifferentColor' for simplicity and clarity 🎨